packages/atomic/src/components/search/facets/atomic-numeric-facet/atomic-numeric-facet.tsx

```diff
@@ -126,6 +127,14 @@ export class AtomicNumericFacet
    */
   @Prop({reflect: true}) public sortCriteria: RangeFacetSortCriterion =
     'ascending';
+  /**
+   * Specifies how a result must match the selected facet values.
+   * Allowed values:
+   * - `atLeastOneValue`: A result will match if at least one of the corresponding facet values is selected.
+   * - `allValues`: A result will match if all corresponding facet values are selected.
+   */
+  @Prop({reflect: true}) public resultsMustMatch: FacetResultsMustMatch =
+    'atLeastOneValue';
```

Comment:
I'm having trouble visualizing a situation where you would want "allValues" for a numeric filter. I think we should have an example here.

Reply:
I agree. Not sure it is relevant to add it to the numeric-facet. 
However, I imagine documents could have a field with multiple numerical values used in a numeric-facet. 
But even with that, I agree that it is not enough to include the option. Let me know what you think